lib/index.js

```diff
@@ -93,4 +93,4 @@ module.exports = function init (options) {
   return new LocalStorage(options);
 };
 
-module.exports.Service = LocalStorage;
+module.exports.MemoryService = MemoryService;
```

Comment:
We should probably leave the export named as `Service`.

Reply:
ok